public/app/features/plugins/extensions/getCoreExtensionConfigurations.ts

```diff
@@ -1,6 +1,7 @@
 import { PluginExtensionAddedLinkConfig } from '@grafana/data';
+import { getDataSourceExtensionConfigs } from 'app/features/datasources/extensions/getDataSourceExtensionConfigs';
 import { getExploreExtensionConfigs } from 'app/features/explore/extensions/getExploreExtensionConfigs';
 
 export function getCoreExtensionConfigurations(): PluginExtensionAddedLinkConfig[] {
-  return [...getExploreExtensionConfigs()];
+  return [...getExploreExtensionConfigs(), ...getDataSourceExtensionConfigs()];
```

Comment:
I think we can remove this (we don't have any core extensions yet if I understand correctly).

Reply:
Roger that. I misunderstood how that array worked. It can all be removed